Source/engine/render/text_render.cpp

```diff
@@ -496,6 +496,11 @@ void OptionLanguageCodeChanged()
 	UnloadFonts();
 	LanguageInitialize();
 	LoadLanguageArchive();
+	effects_cleanup_sfx();

```

Comment:
Unrelated to this PR but  `text_render` is the wrong place for this function, as it makes `text_render` depend on audio. Need to find a better place for it.